examples/provider/resources/notification_policy.tf

```diff
@@ -0,0 +1,6 @@
+resource zitadel_notification_policy notification_policy {
+  depends_on = [zitadel_org.org]
```

Comment:
depends_on is not needed zitadel_org.org.id is referenced below

Reply:
will also remove it on all other examples, as the dependency is defined with the use of attributes of other resources